website/cue/reference/components/sources/gcp_pubsub.cue

```diff
@@ -183,5 +218,22 @@ components: sources: gcp_pubsub: {
 				have the `acknowledgements` setting enabled.
 				"""
 		}
+		auto_concurrency: {
+			title: "Automatic Concurrency Management"
```

Comment:
Thanks for adding this new section 👍 

Just a note that I think you need to have an empty line between paragraphs, if you meant for these to be separated.

Reply:
No, that's intentional, I just put line breaks after each sentence to make editing easier later.